infrastructure/prod.tfvars

```diff
@@ -1,5 +1,6 @@
 vault_section = "prod"
 scheduling_enabled = "true"
 encyption_enabled = "true"
+file_cleanup_enabled = "false"
```

Comment:
no need - default will be used

Reply:
just in case default changes 😱

Reply:
tru dat. i wonder whether we could stick in some smoke test checking the cleanup component is loaded in context or not 🤔. but then it will mean it's running on prod staging slot